docs/manual/developer/03_creating_content.md

```diff
@@ -558,6 +558,61 @@ controls:
       and to use the targeted policy.
 ```
 
+### Reporting status
+
+In some cases, it's useful to know the status of a certain control for a
+specific product. In order to better portray this, it's possible to set
+such information on each control using the `status` key.
+
+The `status` key may hold the following values:
+
+* `pending`: The control is not yet evaluated for the product.
+
+* `not applicable`: The control is not applicable to this product.
+
+* `inherently met`: The control is inherently met by the product.
+
+* `documentation`: The control is addressed by product documentation.
```

Comment:
Out of curiosity, could you point to a control addressed by documentation?

Reply:
https://github.com/ComplianceAsCode/redhat/blob/master/openshift-container-platform-4/policies/SA-System_and_Services_Acquisition/component.yaml#L146-L170

Reply:
I see, I overlooked the narrative as  the controls are marked with `implementation_status: complete`.
So from what I could check, these are used for controls that are about documentation.

Do you imagine `status: documentation` being used for controls that will not have automation and are supposed to be remediated manually? And the control points to documentation.
For example, configuring GRUB or BIOS password.